logic.php

```diff
@@ -1950,6 +1950,10 @@ function keys_vote($raid)
                 'text'          => EMOJI_REFRESH,
                 'callback_data' => $raid['id'] . ':vote_refresh:0'
             ],
+            [
+              'text'          => EMOJI_ALARM1,
```

Comment:
Aren’t 6 buttons in 1 row not a bit too much 🤔 (Refresh, Alarm, I‘m here, Late, Done, Cancel)
Can you provide screenshots?

Reply:
`'text'          => EMOJI_ALARM,`